lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -555,6 +563,7 @@ export class UserAgentApplication {
             }
```

Comment:
I think a log statement would be valuable inside the `else` here.

Reply:
+1

Reply:
Done.